### PR TITLE
VSCode | Return Error when tenant configuration check failed (AST-70028)

### PR DESCRIPTION
**Pull request by AlvoBen**
…error

### CODE REVIEW

File: src/main/wrapper/ExecutionService.ts [hunk 1]
```diff
@@ -153,6 +153,7 @@ export class ExecutionService {
                 logger.info("Exit code received from AST-CLI: " + code);
                 if(code==1){
                     stderr = stdout
+                    reject(stderr)
```
Comment (miryamfoiferCX) [inline src/main/wrapper/ExecutionService.ts:156]: make sure that the rest of the codebase is updated to handle these changes appropriately.

Comment (AlvoBen) [reply src/main/wrapper/ExecutionService.ts:156]: yes it's ok

Comment (greensd4) [inline src/main/wrapper/ExecutionService.ts:156]: Do we have tests to test this behaviour?

Comment (AlvoBen) [reply src/main/wrapper/ExecutionService.ts:156]: No, we do not have any tests in VSCode except UI tests that using mocks.
I think Miryam and Sara will work next sprint on a design for VSCode tests platform

**Review by miryamfoiferCX — APPROVED** [review]

**Pull request merged**